src/tools/events_generator.js

```diff
@@ -238,37 +234,34 @@ EventsGenerator.prototype.generate_audit = function(num, cat) {
     }
 
     const logs_size = events_pool.length;
-    let count = 0;
-    return P.pwhile(() => count < num,
-        () => {
-            count += 1;
-            const chosen = Math.floor(Math.random() * (logs_size));
-            return P.resolve()
-                .then(() => Dispatcher.instance().activity(_.clone(events_pool[chosen])))
-                .then(() => P.delay(1000));
-        });
+    for (let count = 0; count < num; count++) {
```

Comment:
We usually use ++count
```suggestion
    for (let count = 0; count < num; ++count) {
```

Reply:
fix in the rest of the code as well